doc/site/site.gradle

```diff
@@ -66,6 +69,29 @@ dependencies {
     }
 }
 
+node {
+    version = '7.8.0'
+
+    download = true
+
+    nodeModulesDir = file("src/npm")
+}
+
+npmInstall {
+    inputs.file 'src/npm/package.json'
+    outputs.dir 'src/npm/node_modules'
+}
+
+processResources {
+    from(npmInstall) {
+        includeEmptyDirs = false
+        include 'jquery/dist/jquery.min.js'
+        eachFile {
+            it.relativePath = new RelativePath(true, 'public', 'js', it.sourceName)
```

Comment:
ah this is awesome. I was planning to do something like this before I got webpack involved :)